src/main/java/com/solomka/User.java

```diff
@@ -0,0 +1,81 @@
+package com.solomka;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```

Comment:
you can avoid using column annotation if you don't need to change the column name) so, you can remove it out of here. apply this rule everywhere

Reply:
done